ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -224,11 +224,13 @@ measurement:
   current: "Flow of electric charge."
   currentratio: "The ratio of primary amps divided by secondary amps."
   density: "Amount of mass contained in a unit volume."
+  distance: "Linear measurement."
```

Comment:
Please adjust the description to: "Length of measurement between two things."

Reply:
excellent! thanks sir.